Blackbaud.CustomFx.ConstituentPhotoAnalysis.Catalog/ConstituentPhotoAnalysis.Table.xml

```diff
@@ -0,0 +1,42 @@
+﻿<TableSpec
+  xmlns="bb_appfx_table"
+  xmlns:c="bb_appfx_commontypes"
+  ID="9ce8d45a-c025-4832-aebf-613d69574144"
+  Name="Constituent Photo Analysis"
+  Description="Stores information about analyzed constituent photos."
+  Author="Blackbaud Demo"
+  Tablename="USR_CONSTITUENTPHOTOANALYSIS"
+  IsBuiltIn="false"
+  PrimaryKeyAsForeignKeyTablename="CONSTITUENT"
+  >
+
+  <Fields>
+    <EnumField Name="PREDICTEDGENDERCODE" Description="The predicted gender of the constituent." DefaultValue="0" >
+      <EnumValues>
+        <EnumValue ID="0" Translation="Unknown"/>
+        <EnumValue ID="1" Translation="Male"/>
+        <EnumValue ID="2" Translation="Female"/>
+        <EnumValue ID="3" Translation="Other"/>
+      </EnumValues>
+    </EnumField>
+
+    <NumberField Name="ESTIMATEDAGE" Description="The estimated age of the constituent." />
+
+    <DecimalField Name="ANGERRATING" Description="The estimated confidence (0-1) that the person in the photo has an expression of anger." />
```

Comment:
Is the precision of these created Decimal fields enough to hold the ratings coming back from the service?

Reply:
Only 3 digits come back from the service. I added the forth as a safety.